go.mod

```diff
@@ -1,115 +1,74 @@
 module github.com/kedacore/keda
 
-go 1.13
+go 1.13.1
```

Comment:
The go version is supposed to be "go <MAJOR>.<MINOR>" this is producing that I'm not able to use this package as dependency for a controller.

Reply:
@joskfg https://github.com/kedacore/keda/pull/718

Reply:
Thanks a lot!